function/helpers.go

```diff
@@ -37,3 +37,22 @@ func NewJSFetcthOptionArg() JSFetchOptionsArg {
 	}
 	return defaultOptions
 }
+
+type JSSendMailArg struct {
```

Comment:
I wonder if we'd be better just sticking with the `email.SendMailData`. What do you think?

From a "user" point-of-view, might be best to let them have all the option, like the reply-to if they want it from their server-side instead of having a small subset in this JSSendMailArg.